source/common/less/theme-berlin/light.less

```
@@ -211,6 +211,22 @@ body {
         border-bottom-right-radius: 4px;
     }
 
+    .code-block-copy-button{
+      position: absolute;
+      top: 1px;
```

Long code lines may get hidden behind the copy button. Maybe we should move it a bit up:
```suggestion
      top: -15px;
```